src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -530,5 +530,20 @@ private void displayConfidenceLevel()
                 this.lblConfidenceLevel.Margin = new Thickness(left, -15, 0, 0);
             }
         }
+
+        private void zoomScaleLevel_ValueChanged(object sender, RoutedPropertyChangedEventArgs<double> e)
+        {
+            Slider slider = (Slider)sender;
+
+            //Since the percentage goes from 10 to 300, the value is decremented by 10 to standardize. 
+            double percentage = slider.Value - 10;
+            //This is the relation between the margin in pixels and the value of the percentage
+            double marginValue = (79 * percentage / 29) - 480;
+            if (lblZoomScalingValue != null)
```

Comment:
When would this value be null, do you have an example?

Reply:
When the component is initialized, this method is called, but the value of the zoom scale is null.